### DISCUSSION TITLE
询问能否将Frp加入到服务组件中

### DISCUSSION THREAD

**Discussion by novnic** [community/community#39232] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

这是我用过最好的软路由固件，询问能否将Frp加入到服务组件中。

**Comment by jfullstackdev** [comment]
if you can use an online translator and translate this for us, that will be much better

**Comment by queenofcorgis** [comment]
Hi @novnic, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
